Makefile

```diff
@@ -15,7 +15,8 @@ strip: build
 
 .PHONY: build-podman
 build-podman:
-	podman build .
+	ln -f build/Dockerfile Containerfile
```

Comment:
I thought to commit the symlink to git :) Just remove this line, create such symlink and commit it to the git please, thank you!

Reply:
Can you add a comment why is that?